File: frontend/python/visitorcount.py
import requests

def get_visitors():
    url = 'https://i2c9mq7v59.execute-api.us-east-1.amazonaws.com/default/VisitorCounter'
    
    try:
        response = requests.get(url)
        data = response.json()
        
        # Assuming you want to print the data and return it (similar to console.log and return data; in JavaScript)
        print(data)
        return data
        
    except Exception as e:
        print(f"Error occurred: {e}")

# Calling the function to initiate the GET request
get_visitors()

File: frontend/js/visitorcount.js
// GET API REQUEST
async function get_visitors() {
    // call post api request function
    //await post_visitor();
    try {
        let response = await fetch('https://5o3lju0ona.execute-api.us-east-1.amazonaws.com/prod/visitorcount', {
            method: 'GET',
        });
        let data = await response.json()
        document.getElementById("visitors").innerHTML = data['count'];
        console.log(data);
        return data;
    } catch (err) {
        console.error(err);
    }
}


get_visitors();
